src/inputs/checkbox.html

```diff
@@ -0,0 +1,5 @@
+<!--Checkbox input-->
+<script type="text/ng-template" id="wgn-config-input-checkbox">
+	<input type="checkbox"  name="{{ field.id }}" ng-model="editing.config[field.id]">
+	<label class="form-check-label">{{field.name}}</label>
```

Comment:
@wizehive-chrismanjoine minor nitpick, can we put the input inside the label, so you when you click on the text, it selects/deselects the checkbox?

Reply:
The behavior will also be achieved by using `for` on the label and referring it to the input's `id`, assuming it is unique. But I'm with Anna, in that the current input patterns should probably be preferred unless we spot a bug in those patterns that needs adjustment.

Reply:
This has been updated. not sure why I did it that way to begin with.